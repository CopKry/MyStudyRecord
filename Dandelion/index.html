<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Dandelion</title>
    
    <link rel="stylesheet" href="styles/index.css">
</head>
<body>
    <div class="html">
        <header>
            <div class="top wrap clear-hidden">
                <nav class="left ">
                    <ul class="clear-hidden">
                        <li class="left"><a href="#">LIFESTYLE</a></li>
                        <li class="left"><a href="#">ART</a></li>
                        <li class="left"><a href="#">TRAVEL</a></li>
                        <li class="left"><a href="#">FOOD</a></li>
                        <li class="left"><a href="#">FANSHION</a></li>
                        <li class="left"><a href="#">ABOUT</a></li>
                    </ul>
                </nav>
                <div class="search right">
                    <ul class="clear-hidden">
                        <li class="right"><a href="#"><img src="images/search.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/linkedin.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/Instagram.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/pinterest.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/google plus.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/twitter.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/Heart.png" alt=""></a></li>
                        <li class="right"><a href="#"><img src="images/facebook.png" alt=""></a></li>
                    </ul>
                </div>
            </div>
        </header>
        <div class="logo-div wrap">
                <i class="dandelion_logo"></i>
        </div>
        <div class="start-div wrap">
            <img src="images/flower-bg.png" alt="">
            <section class="start-section">
                <p>HOW TO START YOUR</p>
                <p>INDOOR GARDEN</p>
                <p><a href="">NOVEMBER 20.2015</a></p>
                <p>
                    Lorem ipsum dolor sit amet, consectetuer adipiscing elit, 
                    sed diam nonummy nibh euismod tincidunt ut laoreet dolore 
                    magna aliquam erat volutpat. Ut wisi enim ad minim veniam, 
                    quis nostrud exerci tation ullamcorper suscipit lobortis nisl 
                    ut aliquip ex ea commodo consequat. 
                </p>
                <a class="section-last-a" href="">CONTINUE READING</a>
            </section>
        </div>
        <div class="start-div-bottom-div wrap">
            <i class="about-me"><a href=""><p>ABOUT ME</p></a></i>
            <i class="pinterest"><a href=""><p>PINTERSET</p></a></i>
            <i class="recipes"><a href=""><p>RECIPES</p></a></i>
            <a href=""><p class="lifestyle-p">LIFESTYLE</p></a>
        </div>
        <div class="under margin-top-div wrap">
            <p class="title-p">
                Under $100 DIY: Turn Your Closet into 
            </p>
            <p class="title-p title-bottom-p">
                a Major Style Statement
            </p>
            <img src="images/under-img.png" alt="">
            <p class="content">
                    lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat 
                    volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.
                    Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et 
                    accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta 
                    nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Typi non habent claritatem insitam; est usus legentis in iis qui facit eorum claritatem. 
                    Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius.
            </p>
            <div class="div-footer clear-hidden">
                <p class="left">12 COMMENTS</p> <a href="">CONTINUE READING</a> <p class="right">SHARE</p>
            </div>
            <a href=""><p class="lifestyle-p">LIFESTYLE</p></a>
        </div>
        <div class="nine-day wrap margin-top-div">
            <p class="title-p">9 Easy DIY Ideas for Your</p>
            <p class="title-p title-bottom-p">Next Outdoor Party</p>
            <img src="images/nine-day-img.png" alt="">
            <p class="content">
                    orem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat 
                    volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in 
                    hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et 
                    accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta 
                    nobis eleifend option congue nihil imperdiet doming id quod mazim placerat facer possim assum. Typi non habent claritatem insitam; est 
                    usus legentis in iis qui facit eorum claritatem. Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius.
            </p>
            <div class="div-footer clear-hidden">
                <p class="left">12 COMMENTS</p> <a href="">CONTINUE READING</a> <p class="right">SHARE</p>
            </div>
        </div>
        <div class="div-cloud wrap">
            <i class="cloud-i">AD BANNER</i>
        </div>
        <div class="more-div wrap clear-hidden">
            <ul class="left">
                <li><img src="images/orange.png" alt=""></li>
                <li><a href=""><p>FOOD</p></a></li>
                <li><p>How to make lemon shortbread</p></li>
                <li><a href="">READ ARTICLE</a></li>
            </ul>
            <ul class="right">
                <li><img src="images/meat.png" alt=""></li>
                <li><a href=""><p>FOOD</p></a></li>
                <li><p>BEEF CHATEAUBRIAND RECIPE</p></li>
                <li><a href="">READ ARTICLE</a></li>
            </ul>
            <ul class="right">
                <li><img src="images/house.png" alt=""></li>
                <li><a href=""><p>FOOD</p></a></li>
                <li><p>5 PLACES TO VISIT THIS WINTER</p></li>
                <li><a href="">READ ARTICLE</a></li>
            </ul>
            <ul class="left">
                <li><img src="images/lifestyle.png" alt=""></li>
                <li><a href=""><p>LIFESTYLE</p></a></li>
                <li><p>HOW TO BUILD A CREAT WORKSPACE</p></li>
                <li><a href="">READ ARTICLE</a></li>
            </ul>
        </div>
        <div class="load-more wrap">
            <p>LOAD MORE</p>
        </div>
        <div class="news wrap">
            <p>NEWSLETTER</p>
            <p>Join The dandelionite Family! Sign up for our weekly newsletter.</p>
            <div class="email">
                <form action="">
                    <input type="text" placeholder="YOUR E-MAIL ADDRESS">
                    <input type="submit" value="GO">
                </form>
            </div>
        </div>
        <div class="last-bg">
        </div>
        <footer class="wrap clear">
            <p class="right">&copy; 2015 - All Rights Reserved</p>
        </footer>
    </div>
</body>
</html>