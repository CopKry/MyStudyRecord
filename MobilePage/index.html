<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>求职</title>
    <script src="js/rem.js"></script>
    <link rel="stylesheet" href="styles/reset.css">
    <link rel="stylesheet" href="styles/index.css">
</head>

<body>
    <div id="app">
        <!-- 顶部 -->
        <div class="top">
            <div class="top-left">
                <a href=""><i class="friend-icon"></i></a>
                <a href=""><i class="note-icon"></i></a>
            </div>
            <div class="top-right">
                <a href=""><i class="camera-icon"></i></a>
                <a href=""><i class="scan-icon"></i></a>
            </div>
        </div>
        <!-- 搜索区域 -->
        <div class="search-div flex flex-between align-items">
            <div class="search-left flex align-items">
                <a class="flex align-items" href="">杭州
                    <i></i>
                </a>
            </div>
            <form action="">
                <input type="text" placeholder="搜索职位/公司">
                <input type="submit">
            </form>
        </div>
        <!-- 导航菜单栏 -->
        <nav class="nav-menu">
            <ul class="flex flex-between align-items">
                <li><a href="">推荐</a></li>
                <li>
                    <a class="flex align-items" href="">求职<i></i></a>
                </li>
                <li>
                    <a class="flex align-items" href="">招聘<i></i></a>
                </li>
                <li><a href="">平台</a></li>
                <li>
                    <a class="news-a flex align-items" href="">消息<i class="news"></i></a>
                </li>
                <li><a href="">我的</a></li>
            </ul>
        </nav>
        <!-- 内容 -->
        <div class="content-div">
            <div class="left-div">
                <a class="face-a" href="">
                    <i class="user-face"></i>
                    <i class="red-ten"></i>
                </a>
            </div>
            <article class="article-div">
                <header>
                    <h3>李蓝枫</h3>
                </header>
                <section>
                    <p>
                        刚来到杭州不久，想找一份关于行政管理的工作，
                        感兴趣的HR可以查看我的详情，并与我沟通。
                    </p>
                    <video src="videoes/391199_08141a18da2bfd8ce181d6d1da38a3f6_0406ce8dc0c1.mp4" poster="images/indexs/Tom.png"
                        controls>
                    </video>
                </section>
                <footer>
                    <a href="">
                        <i class="comment-icon"></i>
                    </a>
                    <a href="">
                        <i class="like-icon"></i>
                    </a>
                    <a href="">
                        <i class="star-icon"></i>
                    </a>
                    <a href="">
                        <i class="download-icon"></i>
                    </a>
                    <a href="">
                        <i class="more-icon"></i>
                    </a>
                </footer>
                <!-- 回复区 -->
                <div>
                    <div class="content-div comment-div">
                        <div class="left-div">
                            <a class="face-a" href="">
                                <i class="user-face"></i>
                            </a>
                        </div>
                        <article class="article-div">
                            <header>
                                <h3 class="flex flex-between align-items">唐小颖 <span>3-23 10:25</span></h3>
                            </header>
                            <section>
                                <p>
                                    唧唧复唧唧木兰当户织，阿姊闻织声，自挂东南枝。天生我才自游泳，断腿断脚。
                                </p>
                            </section>
                        </article>
                    </div>
                    <div class="content-div comment-div flow-div">
                        <div class="left-div">
                            <a class="face-a" href="">
                                <i class="user-face"></i>
                            </a>
                        </div>
                        <article class="article-div">
                            <header>
                                <h3 class="flex flex-between align-items">纪非白 <span>3-23 10:25</span></h3>
                            </header>
                            <section>
                                <p>
                                    唧唧复唧唧木兰当户织，阿姊闻织声，自挂东南枝。天生我才自游泳，断腿断脚。
                                </p>
                            </section>
                            <footer class="flex flex-end">
                                <a href="">更多</a>
                            </footer>
                        </article>
                    </div>
                </div>
            </article>
        </div>
        <div class="content-div">
            <div class="left-div">
                <a class="face-a" href="">
                    <i class="user-face"></i>
                    <i class="red-ten"></i>
                </a>
            </div>
            <article class="article-div">
                <header>
                    <h3>李蓝枫</h3>
                </header>
                <section>
                    <p>
                        刚来到杭州不久，想找一份关于行政管理的工作，
                        感兴趣的HR可以查看我的详情，并与我沟通。
                    </p>
                    <video src="videoes/391199_08141a18da2bfd8ce181d6d1da38a3f6_0406ce8dc0c1.mp4" poster="images/indexs/Tom.png"
                        controls>
                    </video>
                </section>
                <footer>
                    <a href="">
                        <i class="comment-icon"></i>
                    </a>
                    <a href="">
                        <i class="like-icon"></i>
                    </a>
                    <a href="">
                        <i class="star-icon"></i>
                    </a>
                    <a href="">
                        <i class="download-icon"></i>
                    </a>
                    <a href="">
                        <i class="more-icon"></i>
                    </a>
                </footer>
                <div>
                    <div class="content-div comment-div">
                        <div class="left-div">
                            <a class="face-a" href="">
                                <i class="user-face"></i>
                            </a>
                        </div>
                        <article class="article-div">
                            <header>
                                <h3 class="flex flex-between align-items">唐小颖 <span>3-23 10:25</span></h3>
                            </header>
                            <section>
                                <p>
                                    唧唧复唧唧木兰当户织，阿姊闻织声，自挂东南枝。天生我才自游泳，断腿断脚。
                                </p>
                            </section>
                        </article>
                    </div>
                    <div class="content-div comment-div flow-div">
                        <div class="left-div">
                            <a class="face-a" href="">
                                <i class="user-face"></i>
                            </a>
                        </div>
                        <article class="article-div">
                            <header>
                                <h3 class="flex flex-between align-items">纪非白 <span>3-23 10:25</span></h3>
                            </header>
                            <section>
                                <p>
                                    唧唧复唧唧木兰当户织，阿姊闻织声，自挂东南枝。天生我才自游泳，断腿断脚。
                                </p>
                            </section>
                            <footer class="flex flex-end">
                                <a href="">更多</a>
                            </footer>
                        </article>
                    </div>
                </div>
            </article>
        </div>
        <!-- 结尾 -->
        <footer class="footer-div">
            <ul class="flex flex-between align-items">
                <li>
                    <a href=""><i class="home-icon"></i></a>
                </li>
                <li>
                    <a href=""><i class="page-icon"></i></a>
                </li>
                <li>
                    <a href=""><i class="publish-icon"></i></a>
                </li>
                <li>
                    <a href=""><i class="video-icon"></i></a>
                </li>
                <li>
                    <a href=""><i class="self-icon"></i></a>
                </li>
            </ul>
        </footer>
    </div>
</body>
</html>