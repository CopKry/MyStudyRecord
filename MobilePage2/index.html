<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Chat</title>
    <link rel="stylesheet" href="styles/reset.css">
    <script src="js/rem.js"></script>
</head>
<body>
    <div id="app">
        <div class="top">
            <ul class="flex flex-between align-items">
                <li>
                    <a href=""><img class="back-icon" src="images/Back.png" alt=""></a>
                </li>
                <li>
                    <h1 class="flex flex-column align-items">
                        Erick Scott
                        <small>Online Now</small>
                    </h1>
                </li>
                <li>
                    <a href=""><img class="phone-icon" src="images/phone.png" alt=""></a>
                </li>
            </ul>
        </div>
        <article class="news-div">
            <section class="left-news first-new">
                <p class="time flex flex-end align-items">11.10 <i class="done"></i></p>
                <ul class="flex">
                    <li class="flex flex-column flex-end"><a href=""><img src="images/user_1.png" alt=""></a></li>
                    <li>
                        <video src="videoes/391199_08141a18da2bfd8ce181d6d1da38a3f6_0406ce8dc0c1.mp4" poster="images/Tom.png"
                        controls>
                        </video>
                        <span></span>
                    </li>
                </ul>
            </section>
            <section class="left-news last-news">
                <p class="time flex flex-end align-items">11.53 <i class="done"></i></p>
                <ul class="flex">
                    <li class="flex flex-column flex-end"><a href=""><img src="images/user_1.png" alt=""></a></li>
                    <li>
                        <audio src="musics/you_even_old_S_H_E.mp3" controls></audio>
                        <span></span>
                    </li>
                </ul>
            </section>
            <section class="right-news">
                <p class="time flex flex-end align-items">10.46 <i class="done"></i></p>
                <ul class="flex flex-reverse">
                    <li class="font-news">
                        <p>I thought Iâ€™d take an educated guess as to what a day spent with Wes Anderson in Paris</p>
                        <span></span>
                    </li>
                </ul>
            </section>
        </article>
        <div class="full-none">
        </div>
        <footer class="talk-footer">
            <div class="flex flex-between align-items write-div">
                <form action="">
                <input type="text" placeholder="Write a messafe...">
                </form>
                <a href="">
                    <i class="mico-icon"></i>
                </a>
            </div>
            <div class="more-active flex flex-between align-items">
                <a href=""><i class="font-icon"></i></a>
                <a href=""><i class="smile-icon"></i></a>
                <a href=""><i class="good-icon"></i></a>
                <a href=""><i class="camera-icon"></i></a>
                <a href=""><i class="photos-icon"></i></a>
                <a href=""><i class="share-icon"></i></a>
            </div>
        </footer>
    </div>
</body>

</html>