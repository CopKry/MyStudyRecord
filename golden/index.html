<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>golden</title>
    <link rel="stylesheet" href="styles/index.css">
</head>
<body>
    <div class="all">
        <header>
            <div class="top wrap">
                <h1><img class="logo" src="images/Logo-Treehouse@2x.png" alt="logo-golden"></h1>
                <nav class="top-nav">
                    <ul>
                        <li><a href="#">HOME</a></li>
                        <li><a href="#">SERVICES</a></li>
                        <li><a href="#">PROTFOLIO</a></li>
                        <li><a href="#">ABOUT</a></li>
                        <li><a href="#">CONTACT</a></li>
                    </ul>
                </nav>
            </div>
            <div class="title wrap flex-center">
                <ul>
                    <li class="flex-center">
                        <p class="welcome">Welcome To Our Studio!</p>
                    </li>
                    <li class="flex-center">
                        <p class="nice_to_meet">IT'S NICE TO MEET YOU</p>
                    </li>
                    <li class="flex-center"><a href="#" class="tell_more">TELL ME MORE</a></li>
                </ul>
            </div>
        </header>
        <div class="service div-default">
            <div class="div-margin-default wrap">
                <ul>
                    <li class="flex-center"><p class="div-title">SERVICES</p></li>
                    <li class="flex-center"><p class="div-title-small">Proin iaculis purus consequat sem cure.</p></li>
                </ul>
                <ul class="flex-space-around ul-margin-top">
                    <li>
                        <a href="" class="flex-center"><img src="images/1-E-Commerce@2x.png" alt="E-Commerce"></a> 
                        <p class="p-title p-center p-margin-top-bottom">
                            E-Commerce
                        </p>
                        <p class="p-content p-align-justity ">
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </li>
                    <li>
                        <a href="" class="flex-center"><img src="images/2-ResponsiveWeb@2x.png" alt="Responsive Web"></a>
                        <p class="p-title p-center p-margin-top-bottom">
                            Responsive Web
                        </p>
                        <p class="p-content p-align-justity">
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </li>
                    <li>
                        <a href="" class="flex-center"><img src="images/3-WebSecurity@2x.png" alt="Web Security"></a>
                        <p class="p-title p-center p-margin-top-bottom">
                            Web Security
                        </p>
                        <p class="p-content p-align-justity">
                            Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </li>
                </ul>
            </div>
        </div>
        <div class="protfolio div-default div-color-default">
            <div class="div-padding-default wrap">
                    <ul>
                        <li class="flex-center"><p class="div-title">OUR PORTFOLIO</p></li>
                        <li class="flex-center"><p class="div-title-small">Proin iaculis purus consequat sem cure.</p></li>
                    </ul>
                    <ul class="flex-space-between ul-margin-top">
                        <li class="li-white">
                            <a href=""><img src="images/Ebony.png" alt=""></a>
                            <p class="p-title p-center p-other-top">Ebony & Ivory</p>
                            <p class="div-title-small p-center p-other-bottom">Branding</p>
                        </li>
                        <li class="li-white">
                            <a href=""><img src="images/Smart.png" alt=""></a>
                            <p class="p-title p-center p-other-top">Smart Stationary</p>
                            <p class="div-title-small p-center p-other-bottom">Print Design</p>
                        </li class="li-white">
                        <li class="li-white">
                            <a href=""><img src="images/Clever.png" alt=""></a>
                            <p class="p-title p-center p-other-top">Clever Poster</p>
                            <p class="div-title-small p-center p-other-bottom">Print Design</p>
                        </li>
                    </ul>
                    <ul class="flex-space-between ul-margin-top">
                        <li class="li-white">
                            <a href=""><img src="images/Vinyl.png" alt=""></a> 
                            <p class="p-title p-center p-other-top">Vinyl Record</p>
                            <p class="div-title-small p-center p-other-bottom">Product Mock-Up</p>
                        </li>
                        <li class="li-white">
                            <a href=""><img src="images/Treehouse.png" alt=""></a>
                            <p class="p-title p-center p-other-top">Treehouse Template</p>
                            <p class="div-title-small p-center p-other-bottom">Web Design</p>
                        </li>
                        <li class="li-white">
                            <a href=""><img src="images/Burned.png" alt=""></a>
                            <p class="p-title p-center p-other-top">Burned Logo</p>
                            <p class="div-title-small p-center p-other-bottom">Branding</p>
                        </li>
                    </ul>
            </div>   
        </div>
        <div class="about-us">
            <div class="div-margin-default wrap">
                <ul>
                    <li class="flex-center">
                        <p class="div-title">ABOUT AS</p>
                    </li>
                    <li class="flex-center">
                        <p class="div-title-small">Proin iaculis purus consequat sem cure.</p>
                    </li>
                </ul>
                <ul class="ul-margin-top flex-center ul-relation">
                    <li class="li-position-left">
                        <p class="p-right-align p-title">
                            JULY 2010
                        </p>
                        <p class="p-right-align p-title">
                            Our Humble Beginnings 
                        </p>
                        <p class="p-right-align p-content">
                            Proin iaculis purus consequat sem cure 
                        digni ssim. Donec porttitora entum suscipit 
                        aenean rhoncus posuere odio in tincidunt. Proin 
                        iaculis purus consequat sem cure digni 
                        ssim. Donec porttitora entum suscipit.
                        </p>
                    </li>
                    <li><img src="images/July2010.png" alt=""></li>
                </ul>
                <ul class="flex-center ul-relation">
                    <li class="li-position-right">
                        <p class="p-left-align p-title">
                                JANUARY 2011
                        </p>
                        <p class="p-left-align p-title">
                               Facing Startup Battles
                        </p>
                        <p class="p-left-align p-content">
                                Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt. Proin 
                                iaculis purus consequat sem cure digni 
                                ssim. Donec porttitora entum suscipit aenean 
                                rhoncus posuere odio in tincidunt.
                        </p>
                    </li>
                    <li><img src="images/January2011.png" alt=""></li>
                </ul>
                <ul class="flex-center ul-relation">
                    <li class="li-position-left"> 
                        <p class="p-right-align p-title">
                            DECEMBER 2012
                        </p>
                        <p class="p-right-align p-title">
                            Enter The Dark Days 
                        </p>
                        <p class="p-right-align p-content">
                                Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt. Proin 
                                iaculis purus consequat sem cure digni.
                        </p>
                    </li>
                    <li><img src="images/December2012.png" alt=""></li>
                </ul>
                <ul class="flex-center ul-relation">
                    <li class="li-position-right">
                        <p class="p-left-align p-title">
                            FEBRUARY 2014
                        </p>
                        <p class="p-left-align p-title">
                            Our Triumph 
                        </p>
                        <p class="p-left-align p-content">
                                Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt. Proin 
                                iaculis purus consequat sem cure digni 
                                ssim. Donec porttitora entum suscipit aenean.
                        </p>
                    </li>
                    <li><img src="images/February2014.png" alt=""></li>
                </ul>
                <ul class="flex-center ul-relation">
                    <li class="li-position-center">
                        <p class="p-center-align">Our</p>
                        <p class="p-center-align"> STORY</p>
                        <p class="p-center-align">CONTINUES</p>
                        <p class="p-center-align">...</p>
                    </li>
                    <li><img src="images//Our.png" alt=""></li>
                </ul>
            </div>
        </div>
        <div class="our-amazing-team div-color-default">
            <div class="div-padding-default wrap">
                <ul>
                        <li class="flex-center">
                            <p class="div-title">OUR AMAZING TEAM</p>
                        </li>
                        <li class="flex-center">
                            <p class="div-title-small">Proin iaculis purus consequat sem cure.</p>
                        </li>
                </ul>
                <ul class="flex-space-between ul-margin-top">
                    <li>
                        <img src="images/blackradius.png" alt="">
                        <p class="p-title p-center">Kimberly Thompson</p>
                        <p class="p-content p-center p-bottom-top">Marketer</p>
                        <span class="flex-center">
                            <a href="" class="icon-margin"><img src="images/Twitterus.png" alt=""></a>
                            <a href="" class="icon-margin"><img src="images/Facebookus.png" alt=""></a>
                            <a href=""><img src="images/Pinterestus.png" alt=""></a>
                        </span>
                    </li>
                    <li>
                        <img src="images/blackradius.png" alt="">
                        <p class="p-title p-center">Rico Massimo</p>
                        <p class="p-content p-center p-bottom-top">Coder</p>
                        <span class="flex-center">
                            <a href="" class="icon-margin"><img src="images/Twitterus.png" alt=""></a>
                            <a href="" class="icon-margin"><img src="images/Facebookus.png" alt=""></a>
                            <a href=""><img src="images/Google+us.png" alt=""></a>
                        </span>
                    </li>
                    <li>
                        <img src="images/blackradius.png" alt="">
                        <p class="p-title p-center">Uku Mason</p>
                        <p class="p-content p-center p-bottom-top">Graphic Designer</p>
                        <span class="flex-center">
                            <a href="" class="icon-margin"><img src="images/Twitterus.png" alt=""></a>
                            <a href=""><img src="images/Pinterestus.png" alt=""></a>
                        </span>
                    </li>
                </ul>
                <ul>
                    <li class="amazing-footer">
                        <p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit</p>
                        <p>aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
                    </li>
                </ul>           
            </div>
        </div>
        <div class="more-logo">
            <div class="wrap">
                <ul class="flex-space-between">
                    <li><a href=""><img src="images/Envato.png" alt=""></a></li>
                    <li><a href=""><img src="images/WordPress.png" alt=""></a></li>
                    <li><a href=""><img src="images/Tuts+.png" alt=""></a></li>
                    <li><a href=""><img src="images/Microlancer copy.png" alt=""></a></li>
                </ul>
            </div>
        </div>
        <div class="contact-us">
            <div class="div-padding-default wrap">
                <ul>
                    <li class="flex-center">
                        <p class="contact-us-title div-title">OUR AMAZING TEAM</p>
                    </li>
                    <li class="flex-center">
                        <p class="div-title-small">Proin iaculis purus consequat sem cure.</p>
                    </li>
                </ul>
                <div class="form ul-margin-top">
                    <form action="">
                        <ul class="flex-center">
                            <li class="flex-direction-column">
                                <input type="text" class="text-input" class="text-name" placeholder="YOUR NAME* "> 
                                <input type="text" class="text-input" class="text-email" placeholder="YOUR E-MAIL*">
                                <input type="text" class="text-input" class="text-subject" placeholder="SUBJECT">
                            </li>
                            <li>
                                <textarea name="" id="" cols="30" rows="10" class="message" placeholder="YOUR MESSAGE*"></textarea>
                            </li>
                        </ul>
                        <span class="flex-center">
                            <input type="submit" value="SEND MESSAGE" class="subimt-center">
                        </span>
                    </form>
                </div>
            </div>
        </div>
        <footer>
            <div class="footer wrap flex-center">
                <ul class="flex-center">
                    <li><img src="images/©Copyright 2014 FreebiesXpress.com@2x.png" alt="©Copyright 2014 FreebiesXpress.com@2x"></li>
                    <li><a href="#" class="icon-margin"><img src="images/Twitter.png" alt="Twitter"></a></li>
                    <li><a href="#" class="icon-margin"><img src="images/Facebook.png" alt="Facebook"></a></li>
                    <li><a href="#" class="icon-margin"><img src="images/Pinterest.png" alt="Pinterest"></a></li>
                    <li><a href="#"><img src="images/Google+.png" alt="Google+"></a></li>
                </ul>
            </div>
        </footer>
    </div>
</body>
</html>